server/endpoints/Profile.ts

```diff
@@ -60,7 +72,11 @@ export const countReviewsByStudentId: Endpoint<NetIdQuery> = {
 };
 
 /**
- * [getTotalLikesByStudentId] returns the total number of likes a student has gotten on their reviews
+ * Returns the number of likes that match the reviewIds associated with a given
+ * netId and error if there are none
+ * This method counts the total number of likes received by a student.
```

Comment:
hey Helen, nice work! I think here we should change `"This method counts the total number of likes received by a student."` to  `"This method calculates the total number of likes a student has received on all their reviews."` to be a little more clear